guides/data_and_structure.md

```diff
@@ -0,0 +1,151 @@
+# Data Model and Structure
+
+## Content
+
+#### Hierarchy
+
+- Network
+	- Podcast
+		- Episode
+			- Audio
+	- Audio
+
+
+
+### Network
+
+Field   | Type     | Description 
+------: | :--------| :-----------
+`title` | `String` | The name of the network
+`image` | `Image`  | An avatar style image representing the network
+ 
+ 
+Potential fields open for discussion: 
+
+* `slug` 
+* `link`
+
+### Podcast
+
+A **Podcast** belongs to exactly a **Network**
```

Comment:
Might be a good idea to define exact wording to describe relationships. "_a/an_ entity" is ambiguous. Suggestions:

- X belongs to exactly one Y (1:1, NULL not allowed)
- X belongs to no Y or exactly one Y (1:1, NULL allowed)
- X belongs to at least one Y (1:n, NULL not allowed)
- X belongs to no Y or any number of Y (1:n, NULL allowed)

Reply:
okay, will work that in. Probably should also be put on both sides. currently I opted for just listing the belongs-to sides.